daemon/inspect.go

```diff
@@ -59,7 +66,22 @@ func (daemon *Daemon) ContainerInspect(name string) (*types.ContainerJSON, error
 		StartedAt:  container.State.StartedAt,
 		FinishedAt: container.State.FinishedAt,
 	}
-
+	volumes := make(map[string]string)
+	volumesRW := make(map[string]bool)
+	for _, v := range container.volumes {
+		config := container.VolumeConfig[v.Name()]
+		path, err := v.Mount()
+		if err != nil {
+			return nil, err
+		}
+		volumes[config.Destination] = path
+		volumesRW[config.Destination] = config.RW
+		v.Unmount()
```

Comment:
Yeah, but why we need `Unmount` in inspect? It looks that it'll complicate unmount implementation for all drivers, because I think all supposing that mount and unmount will be called only once during process execution.